test/BenchmarkMiniMax.java

```diff
@@ -0,0 +1,116 @@
+import com.TTT.Board;
+import com.TTT.MiniMax;
+import com.TTT.PlayerMarkers;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class BenchmarkMiniMax {
+
+    private Board board;
+    private MiniMax miniMax;
+
+    @Before
+    public void setUp() {
+        board = new Board(4);
+        miniMax = new MiniMax();
+    }
+
+    @Test
+    public void firstMoveLessThan200ms() {
+        long startTime = System.currentTimeMillis();
+        miniMax.nextMove(board, PlayerMarkers.X);
+        long stopTime = System.currentTimeMillis();
+
+        long totalTime = stopTime - startTime;
+        assertEquals(totalTime, 100, 100);
+    }
+
+    @Test
+    public void averageTimeForEndGameMove() {
+        long startTime = System.currentTimeMillis();
+
+        int i = 0;
+        while(i < 100) {
+            board.placeMarker(0, "X");
```

Comment:
I'm still not sure why I would be ok adding code to make testing easier, as Jim suggested this before. I would have a constructor method just for testing - I thought that was the sort of thing I was supposed to avoid. I do believe you both, I just don't fully understand why you can justify this.